module/actor.js

```diff
@@ -853,6 +858,12 @@ class DCCActor extends Actor {
     // Damage roll
     const damageRollResult = await this.rollDamage(weapon, options)
 
+    // A successful attack always inflicts a minimum of 1 point of damage
+    if (damageRollResult.roll._total <1) {
```

Comment:
We handle this elsewhere, damage rolls less than 1 will be rounded to 1 and the real result shown in brackets.

This possibly only works on the combined attack/damage roll version of the chat card, in which case we should implement an equivalent in the separate chat card flow.

Reply:
Yes this happens only with the Foundry standard chat card.